test/cpp/end2end/xds/xds_override_host_end2end_test.cc

```diff
@@ -440,6 +449,36 @@ TEST_P(OverrideHostTest, ClusterGoneHostStays) {
       GetAffinityCookieHeaderForBackend(DEBUG_LOCATION, 1, kNumEchoRpcs / 3));
 }
 
+TEST_P(OverrideHostTest, DisabledPerPath) {
```

Comment:
In addition to this test, let's also test the case where the config is set at the top level and there is a per-route override that disables it.

Reply:
I will add that in another PR